src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -94,10 +95,19 @@ public void parseCommand_list() throws Exception {
     }
 
     @Test
-    public void parseCommand_listClass() throws Exception {
+    public void parseCommand_add_class() throws Exception {
```

Comment:
Should this be parseCommand_addClass() instead?

Reply:
yep! j edited it hahah